addons/mail/static/src/models/thread/thread.js

```diff
@@ -708,24 +708,37 @@ function factory(dependencies) {
         }
 
         /**
-         * Fetch attachments linked to a record. Useful for populating the store
-         * with these attachments, which are used by attachment box in the chatter.
-         */
-        async fetchAttachments() {
```

Comment:
Keep the method in stable, just make it call the other one with the correct params.